Source/Core/DolphinQt/Settings/SlippiPane.cpp

```diff
@@ -287,13 +287,15 @@ void SlippiPane::ToggleJukebox(bool checked)
 
   if (Core::GetState(Core::System::GetInstance()) == Core::State::Running)
   {
+#ifndef IS_PLAYBACK
     auto& system = Core::System::GetInstance();
     auto& exi_manager = system.GetExpansionInterface();
     ExpansionInterface::CEXISlippi* slippi_exi = static_cast<ExpansionInterface::CEXISlippi*>(
         exi_manager.GetDevice(ExpansionInterface::Slot::B));
 
     if (slippi_exi != nullptr)
       slippi_exi->ConfigureJukebox();
+#endif  // IS_PLAYBACK
```

Comment:
I added this because firstly I dont think jukebox is needed when playback is enabled but also I was getting an error with the EXISlippi files not being found or something.